src/studio/studio.c

```diff
@@ -2332,22 +2332,21 @@ static void blitCursor(Studio* studio)
         s32 sprite = CLAMP(tic->ram->vram.vars.cursor.sprite, 0, TIC_BANK_SPRITES - 1);
         const tic_bank* bank = &tic->cart.bank0;
 
-        tic_point hot = {0};
-
-        if(tic->ram->vram.vars.cursor.system)
-        {
-            bank = &getConfig(studio)->cart->bank0;
-            hot = (tic_point[])
-            {
-                {0, 0},
-                {3, 0},
-                {2, 3},
-            }[CLAMP(sprite, 0, 2)];
-        }
-        else if(sprite == 0) return;
-
+		tic_point hot = {0};
+		hot = (tic_point[])
+			{
+				{0, 0},
+				{3, 0},
+				{2, 3},
+			}[CLAMP(sprite, 0, 2)];
+		
+		if(tic->ram->vram.vars.cursor.system)
+		{
+			bank = &getConfig(studio)->cart->bank0;
+		}
+		
         const tic_palette* pal = &bank->palette.vbank0;
-        const tic_tile* tile = &bank->sprites.data[sprite];
+        const tic_tile* tile = &studio->config->cart->bank0.sprites.data[sprite];
```

Comment:
you always use the cursor from the config cart, this code will not work if I want to change the cursor from the game

Reply:
That's true. I'll look into that.